employees/tests/test_views.py

```diff
@@ -217,3 +215,168 @@ def test_delete_report_view_should_delete_report_on_post(self):
         response = self.client.post(path=self.url)
         self.assertEqual(response.status_code, 302)
         self.assertFalse(Report.objects.filter(pk=self.report.pk).exists())
+
+
+class ReportCustomListTests(TestCase):
+    def setUp(self):
+        task_type = TaskActivityType(pk=1, name="Other")
```

Comment:
Same here about `task_activity` in `ReportFactory`.

Reply:
Done